rainloop/templates/index.php

```diff
@@ -1,3 +1,3 @@
 <div style="box-sizing: border-box; width: 100%; height: 100%; padding: 0px; margin: 0px; background-color: #383c43; position: relative; overflow: hidden;"
-><iframe id="rliframe" style="border: none; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px;" tabindex="-1" frameborder="0"

```

Comment:
I don't know exactly why, but having the iframe with a `position: absolute;` style caused it to not be visible. Seems to work well without it.